composer.json

```diff
@@ -21,7 +21,7 @@
         "symfony/framework-bundle": "~2.3",
         "twig/twig": "~1.12",
         "doctrine/collections": "~1.1",
-        "guzzlehttp/guzzle": "~5.0"
+        "guzzlehttp/guzzle": "~6.2"
```

Comment:
If you make this change you might find it has an incompatibility with some other libraries you use ;).  Please allow for using versions 5 and 6 unless there's a good reason why to drop 5.